src/libraries/Microsoft.Extensions.Logging.Abstractions/gen/LoggerMessageGenerator.Emitter.cs

```diff
@@ -198,15 +198,15 @@ private void GenFields(LoggerMethod lm, string nestedIndentation)
             {
                 foreach (LoggerParameter p in lm.TemplateParameters)
                 {
-                    _builder.AppendLine($"            {nestedIndentation}private readonly {p.Type} _{p.Name};");
+                    _builder.AppendLine($"            {nestedIndentation}private readonly {p.Type} _{p.CodeName};");
```

Comment:
Is this going to work? wouldn't generate a compilation error when using prefix `_@`?

Reply:
@maryamariyan I appreciate if you have a few minutes to take a quick look at this one.

Reply:
Just pushed a commit to resolve this problem.